docs/configuration/configuration.md

```diff
@@ -3601,6 +3601,9 @@ queue_config:
   # Retry upon receiving a 429 status code from the remote-write storage.
   # This is experimental and might change in the future.
   [ retry_on_http_429: <boolean> | default = false ]
+  # If set, any sample that is older than  sample_age_limit given 
+  # will not be sent to the remote storage.
+  [ sample_age_limit: <duration> | default = 0s ]
```

Comment:
we should clarify here that 0s is essentially disabling this feature, ie samples will always be sent regardless of age